src/qtfirebaseauth.cpp

```diff
@@ -217,6 +247,16 @@ void QtFirebaseAuth::onFutureEvent(QString eventId, firebase::FutureBase future)
         {
             qDebug() << this << "::onFutureEvent Verification email sent successfully";
         }
+        else if(eventId == __QTFIREBASE_ID + QStringLiteral(".auth.deleteUser"))
+        {
+            qDebug() << this << "::onFutureEvent Delete user successfully";
+            setSignIn(false);
+        }
+
+        else if(eventId == __QTFIREBASE_ID + QStringLiteral(".auth.resetEmail"))
+        {
+            qDebug() << this << "::onFutureEvent reset email sent successfully";
```

Comment:
Maybe add a signal here to let the programmer sign-out any sessions if he need to.
Signal name suggestion: `passwordResetEmailSent`
(So in QML the signal implementation name would be `onPasswordResetEmailSent`)